app/src/main/java/waiter/RouteDirector.java

```diff
@@ -0,0 +1,54 @@
+package waiter;
+
+public class RouteDirector {
+
+    public Route constructGetSimpleGet(RouteBuilder routeBuilder) {
+        return routeBuilder.newUp()
+                .url("/simple_get")
+                .method("GET")
+                .status("200 OK")
+                .headers("Content-Length:")
+                .body("")
+                .build();
+    }
+
+    public Route constructHeadSimpleGet(RouteBuilder routeBuilder) {
+        return routeBuilder.newUp()
+                .url("/simple_get")
+                .method("HEAD")
+                .status("200 OK")
+                .headers("Content-Length:")
+                .body("")
+                .build();
+    }
+    
+    public Route constructGetSimpleGetWithBody(RouteBuilder routeBuilder) {
+        return routeBuilder.newUp()
+                .url("/simple_get_with_body")
+                .method("GET")
+                .status("200 OK")
+                .headers("Content-Length:")
+                .body("Hello world")
+                .build();
+    }
+
+    public Route constructNoUrl(RouteBuilder routeBuilder) {
+        return routeBuilder.newUp()
+                .url("")
```

Comment:
When and how will this route be used?

Reply:
That seems strange to me that we need to create a route for a route that doesn't exist. Is there a way we can get the same result without doing this?

Reply:
Hmmm, while I agree this could be confusing, I am confused as to how the server will know how to respond to a resource that doesn't exist if I don't have a route for it?

Reply:
Working on status enums now!